io/net/network_test.py

```diff
@@ -75,7 +75,7 @@ def setUp(self):
                                  peer_password=self.peer_password)
         self.peer_interface = self.peerinfo.get_peer_interface(self.peer)
         self.mtu_set()
-        if not self.ping_check("-c 2"):
```

Comment:
Is this a related change ?
If not, can we wait till that is merged and pull latest code before submitting this PR ?